package/com.unity.formats.usd/Documentation~/USD.md

```diff
@@ -1,126 +1,114 @@
-
-To import materials from USD, firstly import the USD file using the USD menu. Then, to get the materials to render in the scene, change the Import Settings > Materials in the Inspector to 'Import Preview Surface'. Finally reimport the USD file.
-
-## Streaming Playback via Timeline
-
-After importing a USD file with either skeletal or point cache animation, open
-the Timeline window. Select the root of the USD file.
-
-Create a playable director by clicking the "Create" button in the Timeline window.
-Next, drag the root USD file into the Timeline to create a track for this object.
-Finally, drag the USD file once more to add a USD clip to the track for plaback.
-Scrubbing through time will now update the USD scene by streaming dat from USD.
-
-Timeline playback is multi-threaded using the C# Job System.
-
-## Variants, Models, & Payloads
-
-Access to variant selection, model details, and payload state are all accessible via
-the inspector on the game object at which these features were authored. Note that Payloads *are not loaded by default*. USD files using Payloads must be reloaded after changing the Payload Policy in the Inspector to 'Load All' for them to appear in the Scene. 
-
-## Exporting USD files
-
-USD files can be created by exporting GameObjects selected in the Hierarchy using the USD context menu options, or using the Recorder.
-
-### Exporting via Recorder
-
-> **Prerequisite:** You need to install the [Recorder package](https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html).  
-
-To export a USD composition via Recorder, you can either use the Recorder Window :
-* From the Editor main menu, select **Window > General > Recorder > Recorder Window**,
-* Then click on **+ Add Recorder**,
-* And select **USD Clip**.
-
-Or add a Recorder track:
-* From the Timeline window, right-click and select **UnityEditor.Recorder.Timeline > Recorder Track**,
-* Then right-click on the track and **add Recorder Clip**,
-* And in the **selected recorder**, choose **USD Clip**.
-
-Using Recorder is the recommended option *when compatibility with runtime is not required*.
-
-### Exporting via Usd Recorder Track (Legacy)
-
-When compatibility with runtime is required (i.e for a standalone build), the recommended option is to use the USD package Recorder Track :
-* From the Timeline window, right-click and select **Unity.Formats.USD > Usd Recorder Track**,
-* Then right-click on the track and **add USD Recorder Clip**.
-
->  **Note:** This feature has no dependency to and is not based on the Recorder package.
-
+* Importing and exporting USD:
+    * Import as GameObject, Prefab, or Timeline Clip
+        * Formats: USDA, USDC, USDZ
+    * Export Game Objects to USD
+        * Formats: USDA, USDC, USDZ
+        * Export of Transform Overrides
+    * Export via Recorder package
+* Composition:
+    * Variant Selection
+    * Payloads
+        * Load All and Individual
+    * Layer stacks
+* Geometry:
+    * UV Set
```

Comment:
I kind of want to specify somewhere in the README that viewing UV channels within Unity is available on Unity 2020+
Just like as a reminder

What do you think?

Reply:
That's not a specification of this package but of USD, so I don't think this is the right place :)